Libraries/zlib/zlib_v1.3^0.txt

```diff
@@ -0,0 +1 @@
+"v1.3^0"
```

Comment:
where does this caret "^" come from?
I would have expected "v1.3.0".

Reply:
That's how the git tag is defined. I write git tag as text file to know from which version of this third party library is copied from.

Reply:
I do not quite understand: This is the tag, right?
https://github.com/madler/zlib/releases/tag/v1.3 and `git tag` simply returns 
```
v0.71
v0.79
v0.8
v0.9
v0.91
v0.92
v0.93
v0.94
v0.95
...
v1.2.6.1
v1.2.7
v1.2.7.1
v1.2.7.2
v1.2.7.3
v1.2.8
v1.2.9
v1.3
```

Reply:
ah I use **git describe --contains HEAD** to the text file, which returns v1.3^0.  When I was using git tag sometimes that was empty, so used git describe https://github.com/gangatp/lib3mf/blob/d1817bebc4a551d53a9494ff7284c87a6cb033bc/submodules/update_libs.sh#L22

Reply:
i see.

Should be fine, I guess